nixos/modules/services/mail/roundcube.nix

```diff
@@ -95,6 +95,14 @@ in
       '';
     };
 
+    maxAttachmentSize = mkOption {
+      type = types.int;
+      default = 18;
+      description = ''
+        The maximum attachment size in MB. Note: The value is not 100% accurate since rounducbe uses a percentage of php upload configuration.
+      '';
```

Comment:
How about adding `apply = x: "${toString (x * 1.3)}M";` here? Then you can always refer to `cfg.maxAttachmentize`.

Reply:
Wasn't aware of that possibility. Changed the code as proposed.